### PR TITLE
add tests for grpc health probes and seccompProfile

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

*Issue #, if available:*

*Description of changes:*
Adding a test to ensure we don't break scenario in https://github.com/aws/amazon-eks-pod-identity-webhook/issues/165 and https://github.com/aws/amazon-eks-pod-identity-webhook/issues/168

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

PS: the last commit missed a file in vendor/ adding it here.

### CODE REVIEW

File: vendor/github.com/emicklei/go-restful/v3/log/log.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package log
```
Comment (nckturner) [inline vendor/github.com/emicklei/go-restful/v3/log/log.go:1]: why is this file included?

Comment (dims) [reply vendor/github.com/emicklei/go-restful/v3/log/log.go:1]: the previous commit this file got left out. (tests passed as i added a "go mod tidy/vendor" explicitly. somehow (gitignore?) this file got left out and i had to do a "git add" by "force" to do this in this PR

**Review by nckturner — APPROVED** [review]

**Review by orsenthil — APPROVED** [review]
LGTM
